### PR TITLE
Adding WB hashing for bls12-381 and bls12-377

### PR DESCRIPTION
**Pull request by drskalman**
## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [x] Targeted PR against correct branch (master)
- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Wrote unit tests
- [x] Updated relevant documentation in the code
- [x] Added a relevant changelog entry to the `Pending` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

**Review by mmagician — COMMENTED** [review]
Looks good 👍🏼

File: bls12_377/src/curves/g2_swu_iso.rs [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+pub const G2_GENERATOR_X_C0: Fq = MontFp!("44471777796618567688228760095584248343372454885978087674329841655595593880133139294404651664057692271364857231527");
+
+/// G2_GENERATOR_X_C1 =
+pub const G2_GENERATOR_X_C1: Fq = MontFp!("152209914092745808277594956866181055187624831129109767937025242463317365117655129123148193049673425418513926319001");
+
+/// G2_GENERATOR_Y_C0 =
+pub const G2_GENERATOR_Y_C0: Fq = MontFp!("115206687171448860889110309021279060303629519187879257051215751573842462972180856243991157572371972099444077110343");
+
+/// G2_GENERATOR_Y_C1 =
+pub const G2_GENERATOR_Y_C1: Fq = MontFp!("191377956145194479040228903677940355038998863371661730030204479850936075480341608934735952709786495341106477933498");
```
Comment (Pratyush) [inline bls12_377/src/curves/g2_swu_iso.rs:100]: Can we directly specify this inline in `G2_GENERATOR_X` and `G2_GENERATOR_Y`?

**Comment by drskalman** [comment]
Pending https://github.com/arkworks-rs/algebra/pull/517

File: bls12_377/src/curves/g2_swu_iso.rs (hunk 1 above)
Comment (drskalman) [reply bls12_377/src/curves/g2_swu_iso.rs]: fixed [here](https://github.com/w3f/arkworks-curves/pull/11/commits/86ec5aae5708921f2040147d8c6fd1e30cc220e4#diff-5f6dece84d541d07eee0ae081e6e8b1c3ba876851c7591391f07bab9591122d0R74) and [here](https://github.com/w3f/arkworks-curves/pull/11/commits/86ec5aae5708921f2040147d8c6fd1e30cc220e4#diff-3658c7f4fe12cc2a0190ce2b0809d509a3bc57f5e9737869593afcc341e94f4cR68)

**Comment by drskalman** [comment]
I have @mmagician oral blessing to merge.

**Comment by burdges** [comment]
IRTF H2C draft says Edwards curves should use Elligator, not WB.  In fact, this means, or should mean, curves for which an Edwards form exists, should use Elligator, not WB.  Ergo, people should really not be using WB for BLS12-377.

**Pull request merged**